lib/grizzly/zwave/commands/schedule_entry_lock_enable_all_set.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Grizzly.ZWave.Commands.ScheduleEntryLockEnableAllSet do
+  @moduledoc """
+  This command enables or disables all schedules for type Entry Lock.
+
+  Params:
+
+    * `:enabled` - Indicates whether the schedule for the user is enabled or disabled
+
+  """
+
+  @behaviour Grizzly.ZWave.Command
+
+  alias Grizzly.ZWave.Command
+  alias Grizzly.ZWave.CommandClasses.ScheduleEntryLock
+
+  @type param :: {:enabled, 0x00 | 0x01}
```

Comment:
I would have it be 
@type param :: {:enabled, boolean}
and then convert true to 0x01 and false to 0x00, and vice-versa